### DISCUSSION TITLE
Replay with offset

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20378] [Q&A] [answered]
Is this a known issue?
Is it only because it's an aspire map?

https://user-images.githubusercontent.com/40531871/191118316-81700bd5-931b-4cb8-a723-6359eeb3f77a.mp4

I was about to check another replay to verify if all replays have offset, and now both my replays have same score? 
![image](https://user-images.githubusercontent.com/40531871/191118477-7d6b9f2f-05ed-4850-a76a-04502666501c.png)
This was when I finished the second replay (the other replay  is old)
![image](https://user-images.githubusercontent.com/40531871/191118561-0420d542-ff37-4c9a-b379-8383e140a2d8.png)

**Comment by smoogipoo** [comment]
- Can you export + provide the replay?
- Do you have any audio offset / universal offset?

**Reply by AuracleTech** [reply]
1. Here's the 2 replays, I changed extension to .zip so GitHub would upload just rename to .osr
[3days Old - Auracle playing Culprate - Acid Rain (Karoo13) [Aspire].zip](https://github.com/ppy/osu/files/9612282/3days.Old.-.Auracle.playing.Culprate.-.Acid.Rain.Karoo13.Aspire.zip)
[1day old  - Auracle playing Culprate - Acid Rain (Karoo13) [Aspire].zip](https://github.com/ppy/osu/files/9612285/1day.old.-.Auracle.playing.Culprate.-.Acid.Rain.Karoo13.Aspire.zip)
2. Zero universal, -5.7 on local but might be differ per replay

Note : Depending on which replay I click on different stats shows up

https://user-images.githubusercontent.com/40531871/191396549-d940ac09-ff15-42e5-8338-fe8d53cba229.mp4

**Comment by JacksonChen666** [comment]
it's the [(aspire) map](https://osu.ppy.sh/beatmapsets/948643#osu/1981090) doing weird things like having a 35787:53 intro time (35787 minutes and 53 seconds, or 24 days, 20 hours, 27 minutes and 53 seconds)

![osu_2023-01-03_12-58-34](https://user-images.githubusercontent.com/50755746/210353258-6c2865bf-b391-40a4-ab74-0298135b251f.png)

(autoplay was fine if you're wondering)

same thing happened to my replay, it is extremely offset.
i don't have any global/beatmap offset set.

[jacksonchen666 playing Culprate - Acid Rain (Karoo13) [Aspire].osr.zip](https://github.com/ppy/osu/files/10336337/jacksonchen666.playing.Culprate.-.Acid.Rain.Karoo13.Aspire.osr.zip)

logs (warning: 3 hours of logs):
[runtime.log](https://github.com/ppy/osu/files/10336341/runtime.log)
[performance.log](https://github.com/ppy/osu/files/10336342/performance.log)
[performance-update.log](https://github.com/ppy/osu/files/10336343/performance-update.log)
[performance-input.log](https://github.com/ppy/osu/files/10336344/performance-input.log)
[performance-draw.log](https://github.com/ppy/osu/files/10336345/performance-draw.log)
[network.log](https://github.com/ppy/osu/files/10336346/network.log)
[input.log](https://github.com/ppy/osu/files/10336347/input.log)
[database.log](https://github.com/ppy/osu/files/10336348/database.log)
[performance-audio.log](https://github.com/ppy/osu/files/10336349/performance-audio.log)

**Reply by JacksonChen666** [reply]
here's a replay where i put osu! in the background for like... 5 to 6 hours?

[jacksonchen666 playing Culprate - Acid Rain (Karoo13) [Aspire] (1).osr.zip](https://github.com/ppy/osu/files/10338723/jacksonchen666.playing.Culprate.-.Acid.Rain.Karoo13.Aspire.1.osr.zip)

first couple of hours is me working on stuff, then mostly nothing as i leave the computer (mouse at bottom left because macos hot corner user), then i accidentally skip to the main gameplay by pressing the play button in the "now playing" overlay (F6).

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/22086.
